tests/environment_var/main.tf

```diff
@@ -0,0 +1,18 @@
+module "stdout" {
+  source = "../.."
+
+  command = "echo $foo $bar"
+  environment_variables = {
+    foo = "bar"
+    bar = "baz"
```

Comment:
can you uppercase another one of these? `BAR = "BEER"`

Reply:
can do